pkgs/development/python-modules/thinc/default.nix

```diff
@@ -71,6 +71,10 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
+  postPatch = ''
+    substituteInPlace setup.cfg --replace "blis>=0.7.8,<0.8.0" "blis"
```

Comment:
Note you can often use `pythonRelaxDepsHook` to relax or remove a dependency.

Reply:
I tried that. but it doesn't seem to affect setup.cfg.

Reply:
Please don't do this. BLIS 0.9.x is broken.

https://github.com/explosion/thinc/issues/771#issuecomment-1255825935
https://github.com/NixOS/nixpkgs/pull/207070

Reply:
thanks for spotting this. I will close it now.

Reply:
@danieldk, shall we revert blis to  version 0.7.9, so it is breaking spacy?

Reply:
https://github.com/NixOS/nixpkgs/pull/223722